### DISCUSSION TITLE
Can the screensaver be disabled?

### DISCUSSION THREAD

**Discussion by stalakerob** [Blackymas/NSPanel_HA_Blueprint#1707] [General]
Is there a way to disable the screensaver? Dimming down the home page instead after a while.

**Comment by kroessig** [comment]
go to devices --> esphome --> your panel and set  "Timeout Sleep" to 0,0s.
![image](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/139384422/68e6d4de-6d2a-479b-b137-5be602dd5bbd)

**Reply by stalakerob** [reply]
Great, thanks.

**Comment by irvintim** [comment]
Is it possible to update those values in automation?  So, for instance, change the Timeout Sleep to 0 when Motion is detected in the room, but if the room is dark and no one is around, have an HA automation trigger that sets the sleep timeout to 15 seconds....
